earthmover/operations/groupby.py

```diff
@@ -172,3 +173,5 @@ def _get_agg_lambda(agg_type: str, column: str = "", separator: str = ""):
         }
         return agg_lambda_mapping.get(agg_type)
 
+def _quote_list_items(str_list):
```

Comment:
To handle escaping, we should consider using `json.dumps()` on the string list instead of manually wrapping the items in quotes.

Reply:
Great catch! Had to do this a slightly different way but I added a test case for this and it seems to function identically to the old way without hardcoding any characters!